internal/security/secretstore/init_test.go

```diff
@@ -78,6 +79,52 @@ func TestSaveInitResponse(t *testing.T) {
 	fileOpener.AssertExpectations(t)
 }
 
+func TestGetKnownSecretsToAdd(t *testing.T) {
+	defer os.Clearenv()
```

Comment:
@lenny-intel  this screwed the other test relies on the env $PATH for "touch" so that is why the other unit test is broken

Reply:
Ahh, ok yup. That would do it. Thanks!